Tests/TuistKitAcceptanceTests/HashAcceptanceTests.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import ServiceContextModule
+import TuistAcceptanceTesting
+import XCTest
+
+final class HashAcceptanceTestXcodeProjectiOSFramework: TuistAcceptanceTestCase {
+    func test_xcode_project_ios_framework() async throws {
+        try await ServiceContext.withTestingDependencies {
+            try await setUpFixture(.custom("xcode_project_ios_framework"))
```

Comment:
we usually tended to create explicit cases for these, but I don't hate the idea of using strings directly. I don't think the enum of fixtures brought any value really. Would probably then make sense to just be able to pass a string directly, though:
```swift
try await setUpFixture("xcode_project_ios_framework")
```

Reply:
I couldn't agree more. This is a good example where the value the compiler brings is little compared to the cost of inconvenience of having to add new cases for every new fixture. I'll extend the interface to support passing a string.